website/templates/public/pages/support.mako

```diff
@@ -61,12 +61,17 @@
             </div>
         </div>
         <hr>
+
         <div class="row m-b-lg">
-            <div class="col-sm-6">
+            <div class="col-sm-4">
                 <h5 class="m-t-md f-w-xl"> Do you have Prereg Challenge related questions? </h5>
                 <p>Check out our <a href="https://cos.io/prereg/">Prereg section</a> on the cos.io website. </p>
             </div>
-            <div class="col-sm-6">
+            <div class="col-sm-4">
+                <h5 class="m-t-md f-w-xl"> Are you experiencing downtime with our services? </h5>
+                <p> Check out our <a href="https://status.cos.io"> status page</a> for updates on how our services are operating.</p>
```

Comment:
Looks like you'll end up with an extra space here (since there's one after "our" and another before "status").